po/sv.po

```diff
@@ -0,0 +1,821 @@
+# Swedish translation for MultiVNC.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the MultiVNC package.
+# Åke Engelbrektson <eson@svenskasprakfiler.se>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: MultiVNC\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-05-23 07:37+0200\n"
+"PO-Revision-Date: 2023-05-24 09:20+0200\n"
+"Last-Translator: Åke Engelbrektson <eson@svenskasprakfiler.se>\n"
+"Language-Team: \n"
+"Language: sv\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.1.1\n"
+
```

Comment:
Please also add a `translator-credits` entry as in https://github.com/bk138/multivnc/blob/master/po/es.po#L21 so your name automatically shows up in the About dialog.